pkg/test/resourcefixture/testdata/basic/bigqueryconnection/v1alpha1/bigqueryconnectionconnection/cloudspannerconnectionbasic/update.yaml

```diff
@@ -24,6 +24,6 @@ spec:
     databaseRef:
       name: spannerdb-${uniqueId}
     maxParallelism: 150
-    useDataBoost: True
-    useParallelism: True
-    databaseRole: "admin"
+    useDataBoost: False
+    useParallelism: False
+    databaseRole: "user"
```

Comment:
the previous update.yaml is the same as the create.yaml. This change verifies the MockGCP update now works.